addons/account/views/account_reconcile_model_views.xml

```diff
@@ -210,6 +210,7 @@
                                                     attrs="{'invisible': [('match_partner', '=', False)]}"/>
                                                 <field name="match_partner_ids" nolabel="1"
                                                     widget="many2many_tags"
+                                                    options="{'no_quick_create': True}"
```

Comment:
why does not match_partner_category_ids have one? (below)  And then for the counterpart  entities?  Has this been rebased recently?

Reply:
You can quick create a category, but quick creating partner is not a good practice, (the user can still use create and edit here)

Reply:
Seems weird to allow the one and not the other, because the most normal place would be to create the partner categories from the partner.

Reply:
The task was about preventing the users from quick creating where quick create is not actually happening. When the user clicks on *quick create* on partner, the form view of partner is opened, so it acts exactly like *create and edit*, it doesn't make sense to have two items doing the same thing, however quick creating category does not open a form view and if the user clicks on create and edit the for is opened.